### DISCUSSION TITLE
Getting http error 503 for any request

### DISCUSSION THREAD

**Discussion by bubbleguuum** [trakt/trakt-api#385] [Q&A] [answered]
My Android app which had integrated Trakt for a few years now is getting 503 errors on any request that uses a proper and fresh access token.
Here's a log, invoking /users/settings just after getting a refreshed access_token that I verified is properly passed to "Bearer":

```
I   39:59.023  --> GET https://api.trakt.tv/users/settings h2
                        I   39:59.023  Host: api.trakt.tv
                        I   39:59.023  Connection: Keep-Alive
                        I   39:59.023  User-Agent: okhttp/4.10.0
                        I   39:59.023  Content-Type: application/json
                        I   39:59.024  Accept-Encoding: identity
                        I   39:59.024  trakt-api-key: <REDACTED>
                        I   39:59.024  trakt-api-version: 2
                        I   39:59.024  Authorization: Bearer <REDACTED>
                        I   39:59.024  --> END GET
                        I   39:59.143  <-- 503 https://api.trakt.tv/users/settings (119ms)
```

Same thing happens if I execute that request via the web API test interface.

I do not know when exactly this happened, and it could have been quite a while since Trakt is a very optional feature of my app and rarely used and a user just informed me about it.

**Comment by tysonkerridge** [comment, marked as answer]
The API is down at the moment. You can find details in the below links:

https://status.trakt.tv

https://twitter.com/trakt/status/1604114567579111425?s=46&t=3eKIEWS-aYzODaq-_rkmGQ

https://twitter.com/trakt/status/1602509838713589760?s=46&t=3eKIEWS-aYzODaq-_rkmGQ

**Comment by bubbleguuum** [comment]
Many thanks for that info ! 
I tried everything I could think of but did not thought once the API could be down temporarily.
